src/frontend/src/views/Data/tableHeaders/commonSampleHeaders.tsx

```diff
@@ -9,9 +10,8 @@ export const PRIVATE_ID_HEADER: Header = {
   },
 };
 
-export const PUBLIC_ID_HEADER: Header = {
+export const PUBLIC_ID_HEADER: TableHeader<Sample> = {
```

Comment:
Are these header definitions used for the new table?

Reply:
Good point. This is primarily for exported metadata. I should probably rename the type (we will still need it for the export script), but we can remove the tooltips from the type since it lives in the cell def now.